pkg/quota/api/v1/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package v1 is the v1 version of the API.
+// +genconversion=true
```

Comment:
@mfojtik mind fixing this in your clientset generation PR? You're putting that comment in `types.go` rather than package doc.